docs/fundamentals/networking/sockets/tcp-classes.md

```diff
@@ -78,10 +78,71 @@ The preceding C# code:
 - Writes the sent message to the console.
 - Finally, calls the <xref:System.Net.Sockets.TcpListener.Stop%2A> method to stop listening on the port.
 
+## Equivalent usages of `TcpClient` and `TcpListener` classes in `Socket` class
+
+`TcpListener` and `TcpClient` classes directly uses `Socket` class in underlying implementation. It means you can do everything with `Socket` that you can do with `TcpClient` and `TcpListener`.
+
+### `TcpClient` Constructors
+
+`TcpClient's` default constructor basically tries to create a `dual mode socket`, if it's not viable, it creates IPv4 Socket via using `Socket(SocketType, ProtocolType)` constructor. It means following code snippets are equivalent:
+
+```csharp
+TcpClient client = new TcpClient();
+```
+
+```csharp
+Socket socket = new Socket(SocketType.Stream, ProtocolType.Tcp);
+```
+
+One of `TcpClient's` constructor takes `AddressFamily` enum as parameter. You can pass three different enum value to this constructor, otherwise it throws an <xref:System.ArgumentException>. Valid enums are:
```

Comment:
Can we be more specific and link the mentioned constructor in the form: *"the constructor [`TcpClient(AddressFamily)`](https://review.learn.microsoft.com/en-us/dotnet/api/system.net.sockets.tcpclient.-ctor?view=net-6.0&branch=main#system-net-sockets-tcpclient-ctor(system-net-sockets-addressfamily))"*.
@IEvangelist how can we do it with `xref`?

Reply:
Yes, I prefer xref's. We have tooling locally that resolves them, the syntax is otherwise a bit clunky when suggesting changes from the GitHub UI. Ideally, the first time we refer to a type or member, we should use an xref, then after that, we can use the code syntax.